build/assembly/server.xml

```diff
@@ -0,0 +1,34 @@
+<assembly>
+  
+  <id>server</id>
+  
+  <formats>
+    <format>zip</format>
+  </formats>
+  
+  <includeBaseDirectory>true</includeBaseDirectory>
+  <baseDirectory>${dir.server}</baseDirectory>
+
+  <fileSets>
+    <fileSet>
+        <directory>${project.build.directory}/${dir.jboss-orig}</directory>
+        <outputDirectory>/</outputDirectory>      
+        <includes>
+          <include>**/*.sh</include>
+        </includes>   
+        <fileMode>755</fileMode>  
+    </fileSet> 
+    <fileSet>
+        <directory>${project.build.directory}/${dir.jboss-orig}</directory>
+        <outputDirectory>/</outputDirectory>      
+        <excludes>
+          <exclude>**/*.sh</exclude>
+        </excludes>      
+    </fileSet>
+    <fileSet>
+        <directory>${project.build.directory}/teiid-${project.version}-jboss-dist</directory>
```

Comment:
Pulls in unpacked Teiid jboss dist from filesystem that was created earlier in build.